index.md

```diff
@@ -1,37 +1,3 @@
-## Welcome to GitHub Pages
+## Welcome to Teradata Presto GitHub Pages
```

Comment:
maybe we should add a subtitle or a paragraph like:
"This is a blog space for the Teradata Presto engineering team to share our thoughts and findings on the various software engineering topics we come across during the development of Presto - the open source distributed analytical SQL engine (https://github.com/prestodb/presto). Topics areas include, but are not limited to: Java performance tuning, database internals, SQL optimizers, etc"